src/DataSource/Http.hs

```diff
@@ -150,6 +150,7 @@ postTicketComment
 postTicketComment ZendeskResponse{..} = do
     cfg <- ask
 
+    let ticketTags = TicketTags (renderTicketStatus AnalyzedByScriptV1_1 : getTicketTags zrTags)
```

Comment:
Shouldn't this work and not override the tags?

Reply:
Well I guess how Zendesk API works is that if we post list of tags to the ticket, it'll override the existing ones..

Reply:
Additional notes.
Okay when I fetch the analysis result, I do see some of the tags not being overwritten, which is good but strange.. I'll look into this today.